src/console_cmds.cpp

```diff
@@ -1049,6 +1049,55 @@ DEF_CONSOLE_CMD(ConNewGame)
 	return true;
 }
 
+DEF_CONSOLE_CMD(ConNewHeightMapGame)
+{
+	if (argc == 0) {
+		IConsoleHelp("Load a game by name or index. Usage: 'newheightmapgame [<file | number> [seed]]'");
+		IConsoleHelp("The server can force a new game using 'newheightmapgame'; any client joined will rejoin after the server is done generating the new game.");
+		return true;
+	}
+
+	bool heightmap_loaded = true;
+
+	if (argc >=1 && argc <= 3) {
```

Comment:
argc >=1 is redundant, and the argc <= 3 should be as well.

Reply:
This condition has vanished as part of your advice in another review.